apps/bankkonto-service/build.gradle

```diff
@@ -0,0 +1,89 @@
+plugins {
+    id 'java'
+    id "org.sonarqube" version "3.3"
+    id 'org.springframework.boot' version "2.5.7"
+    id 'io.spring.dependency-management' version "1.0.11.RELEASE"
+}
+
+sonarqube {
+    properties {
+        property "sonar.sourceEncoding", "UTF-8"
+        property "sonar.projectKey", "navit_testnorge_bankkonto_service"
+        property "sonar.projectName", "testnav-bankkonto-service"
```

Comment:
Kan du at projectKey og projectName er det samme? Blir litt lettere når man setter det opp.

Reply:
"testnav-bankkonto-service" for begge er bra nok :)